src/main/java/com/example/onlinebookstore/dto/cart/AddToCartRequestDto.java

```diff
@@ -1,12 +1,12 @@
 package com.example.onlinebookstore.dto.cart;
 
-import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
 import lombok.Data;
 
 @Data
 public class AddToCartRequestDto {
-    @NotNull
+    @Positive
```

Comment:
`NotNull` should be present too because `Positive` considers nulls as valid cases.